src/provider/utils/async.ts

```diff
@@ -1,10 +1,19 @@
 import {Signal} from 'openfin-service-signal';
 
 import {DeferredPromise} from '../common/DeferredPromise';
+import {Timeouts} from '../constants';
+
+export enum CollateApiCallResultsResult {
```

Comment:
Can we call this something more concise? `CollateResult`? `ApiResult`?..

Reply:
Renamed the function to `CollateClientCalls`, so this is naturally a bit shorter